netbsd/NetBSDProcessList.c

```diff
@@ -49,23 +50,64 @@ static char const *freqSysctls[] = {
    NULL
 };
 
+static void NetBSDProcessList_updateCPUcount(ProcessList* super) {
+   NetBSDProcessList* opl = (NetBSDProcessList*) super;
+   const int nmib[] = { CTL_HW, HW_NCPU };
+   const int mib[] = { CTL_HW, HW_NCPUONLINE };
+   int r;
+   unsigned int value;
+   size_t size;
+   bool change = false;
+
+   size = sizeof(value);
+   r = sysctl(mib, 2, &value, &size, NULL, 0);
+   if (r < 0 || value < 1) {
+      value = 1;
+   }
+
+   if (value != super->activeCPUs) {
+      super->activeCPUs = value;
+      change = true;
+   }
+
+   size = sizeof(value);
+   r = sysctl(nmib, 2, &value, &size, NULL, 0);
```

Comment:
Comment what's queried to ease following along.

Reply:
Since `nmib` is now documented in the above lines, should I add more info on the sysctl(3) call?